src/CLClient.ts

```diff
@@ -253,9 +256,15 @@ export class CLClient {
         return this.send('GET', this.MakeURL(apiPath))
     }
 
-    public GetLogDialogs(appId: string, packageIds: string[]): Promise<CLM.LogDialogList> {
+    public GetLogDialogs(appId: string, packageIds: string[], continuationToken?: string, maxPageSize?: string): Promise<CLM.LogQueryResult> {
```

Comment:
Why is maxPageSize a string instead of a number ?

Reply:
nice catch